examples/zcu208_clksynth/create_vivado.tcl

```diff
@@ -0,0 +1,145 @@
+# Copyright 2023 The Aerospace Corporation
+#
+# This file is part of SatCat5.
+#
+# SatCat5 is free software: you can redistribute it and/or modify it under
```

Comment:
In the future we can dispense with the typical copyright notice and replace it with a SPDX identifier. That way the nonsense at the top just becomes:
```python
# Copyright 2016-2023 The Aerospace Corporation
# This file is a part of SatCat5
# SPDX-License-Identifier: LGPL-3.0-or-later
```

Reply:
I agree the license boilerplate is verbose, but that's [how FSF recommends](https://www.gnu.org/licenses/gpl-howto.html) doing it.